### PR TITLE
Basic Database Setup

### PR DESCRIPTION
**Pull request by hadleyac**
I created a basic database method for seeding the database. I also created a method for getting all items from the database. I created corresponding GET and POST server methods. The database was seeded with data from the spreadsheet.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    price, 
+    category,
+    images
+  }).save()
+}
+
+const getAll = () => {
+  const query = Item.find({})
+  const promise = query.exec()
+  return promise;
```
Comment (rcalimlim) [inline database/index.js]: I believe you can instead do this as well if you want the Promise.

`return Item.find({}).exec();`

 However, I returned just the `Query` instance and was fine.

File: server/index.js [hunk 2]
```diff
@@ -15,4 +18,25 @@ app.get("/api", (req, res) => {
...
+});
+
+
+//Seed items database
+app.post('/api/items/seed', (req, res) => {
+
+  console.log(req.body.data)
+  console.log(req.body.data.constructor)
+
+  Promise.all(req.body.data.map(jsonItem => {
```
Comment (rcalimlim) [inline server/index.js]: Not sure if you're actually going to use this in the real thing, but you can abstract some of this with
[`Model.insertMany()`](https://mongoosejs.com/docs/api/model.html#model_Model.insertManyl)

File: database/index.js (hunk 1 above)
Comment (hadleyac) [reply database/index.js]: Good suggestion! I made that change

File: server/index.js (hunk 2 above)
Comment (hadleyac) [reply server/index.js]: Nice! I just did that and it saved a lot of code.

**Pull request closed without merging**